types/tools/tool-settings.d.ts

```diff
@@ -28,51 +27,6 @@ export interface ToolboxConfigEntry {
   data?: BlockToolData
 }
 
-/**
- * Represents single interactive (non-separator) Tunes Menu item
- */
-export type TunesMenuConfigDefaultItem = PopoverItemDefaultParams & {
-  /**
-   * Tune displayed text.
-   */
-  title?: string;
-
-  /**
-   * Tune displayed text.
-   * Alias for title property
-   * 
-   * @deprecated - use title property instead
-   */
-  label?: string
-
-  /**
-   * Menu item parameters that should be applied on item activation.
-   * May be used to ask user for confirmation before executing menu item activation handler.
-   */
-  confirmation?: TunesMenuConfigDefaultItem;
-}
-
-/**
- * Represents single separator Tunes Menu item
- */
-export type TunesMenuConfigSeparatorItem = PopoverItemSeparatorParams;
-
-/**
- * Represents single Tunes Menu item with custom HTML contect
- */
-export type TunesMenuConfigHtmlItem = PopoverItemHtmlParams;
-
-/**
- * Union of all Tunes Menu item types
- */
-export type TunesMenuConfigItem = TunesMenuConfigDefaultItem | TunesMenuConfigSeparatorItem | TunesMenuConfigHtmlItem;
-
-/**
- * Tool may specify its tunes configuration
- * that can contain either one or multiple entries
- */
-export type TunesMenuConfig = TunesMenuConfigItem | TunesMenuConfigItem[];
```

Comment:
its a breaking change, you need to add a note about type renaming to a changelog

Reply:
Returned back TunesMenuConfig with `@deprecated` mark